src/ddspdrum/module.py

```diff
@@ -186,8 +193,15 @@ class VCO(SynthModule):
     >>> two_8ve_chirp = myVCO(np.linspace(0, 1, 1000, endpoint=False))
     """
 
-    def __init__(self, midi_f0: float = 10, mod_depth: float = 50, phase: float = 0):
-        super().__init__()
+    def __init__(
+        self,
+        midi_f0: float = 10,
+        mod_depth: float = 50,
```

Comment:
@maxsolomonhenry what is the upper bound on mod_depth? Surely it can't be infinity?

Reply:
Currently mod depth is in semitones. So the upper bound is 127, assuming osc midi_f0 is 0. 

Otherwise upper bound is (127 - osc midi_f0)

Reply:
Okay great.

Reply:
@maxsolomonhenry what why is mod_depth >= 0? Why can't you also modulate it down?

Reply:
I don't know if you want to get into this now, or just make a note of it.

Reply:
I suppose you could, sure. Would have another bound, obviously. -- but then we've have to update the SquareSaw Oscillator... k calculation.

Reply:
I don't think it needs to be changed, it looks fine?

```
        # What does k mean here? Can we give it a better name?
        f0 = midi_to_hz(self.midi_f0 + self.mod_depth)
        return 12000 / (f0 * np.log10(f0))
```

Or is mod_depth just non-negative by convention?

What is `k` anyway? Is there a better name for it?